iquip/apps/monitor.py

```diff
@@ -433,53 +434,54 @@ def profile_info(
     # frequency info
     if frequency_info is None:
         frequency_info = {}
-    frequency_info["ndecimals"] = frequency_info.get("ndecimals", 2)
-    frequency_info["min"] = frequency_info.get("min", 0)
-    frequency_info["max"] = frequency_info.get("max", 4e8)
-    unit = frequency_info.get("unit", "Hz")
+    frequency_info.setdefault("ndecimals", 2)
+    frequency_info.setdefault("min", 1e6)
+    frequency_info.setdefault("max", 4e8)
+    unit = frequency_info.setdefault("unit", "Hz")
     if unit not in ["Hz", "kHz", "MHz"]:
-        logger.warning("The unit of frequency, %s is invalid.", unit)
-        unit = "Hz"
-    frequency_info["unit"] = unit
-    frequency_info["step"] = frequency_info.get("step", 1)
+        frequency_info["unit"] = "Hz"
+        logger.warning("The unit of frequency, %s is invalid."
+                       "Hence, the unit is set to Hz.", unit)
```

Comment:
This will not make a space: "... invalid.Hence, ...".
BTW, I think it is okay to write just necessary information briefly for logs.
For example: `"Invalid frequency unit: %s (set to Hz)."`

Reply:
> This will not make a space: "... invalid.Hence, ...".
Oh, I intended to insert one emtpy space, but I made a mistake😅

Reply:
> BTW, I think it is okay to write just necessary information briefly for logs.
For example: "Invalid frequency unit: %s (set to Hz)."

It looks more compact! I will apply it.